docs/.vitepress/config.ts

```diff
@@ -162,6 +162,25 @@ export default defineConfig({
             { text: 'Mouse Bindings', link: '/reference/mouse_bindings.md' },
             { text: 'Menus', link: '/reference/menus.md' },
             { text: 'Metadata', link: '/reference/metadata.md' },
+            {
+              text: 'Package Control',
+              items: [
+                { text: 'Channels and Repositories', link: '/reference/pc_channels_repositories.md' },
```

Comment:
Is it not worth dropping these another folder deep? `/reference/package-control/channels-repositories.md`? (I also changed to kebab-case because SEO likes that better.

Reply:
Makes sense!

Reply:
> I also changed to kebab-case because SEO likes that better

I am matching the surrounding existing files names now though. Shall I move those too (that's an SEO no-no of course... 🤷🏻‍♂️ )?

Reply:
Can we 301 them for a month or two?